docs/sources/mimir/release-notes/v2.11.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Grafana Mimir version 2.11 release notes"
+menuTitle: "V2.11 release notes"
+description: "Release notes for Grafana Mimir version 2.11"
+weight: 1000
+---
+
+# Grafana Mimir version 2.11 release notes
+
+Grafana Labs is excited to announce version 2.11 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Sampled logging of errors in the ingester.** A high-traffic Mimir cluster can occasionally become bogged down logging high volumes of repeated errors. You can now reduce the amount of errors outputted to logs by setting a sample rate via the `-ingester.error-sample-rate` CLI flag.
+- **Add total request size instance limit for ingesters.** This limit protects the ingesters against requests that together may cause an OOM. Enable this feature by setting the `-ingester.instance-limits.max-inflight-push-requests-bytes` CLI flag.
+- **Reduce the resolution of incoming native histograms samples** if the incoming sample has too many buckets compared to `-validation.max-native-histogram-buckets`. This is enabled by default but can be turned off by setting the `-validation.reduce-native-histogram-over-max-buckets` CLI flag to `false`.
+- **Include a `Retry-After` header in recoverable error responses from the distributor.** This can protect your Mimir cluster from clients that default to retrying very quickly. Enable this feature by setting the `-distributor.retry-after-header.enabled` CLI flag.
+- **Improved query-scheduler performance under load** by using a central dispatcher to allocate requests to queriers instead of having each querier spend cycles checking for work to do on each incoming request.
```

Comment:
I think the description of this is a bit too in the weeds for most users - how about something like this?

```suggestion
- **Improved query-scheduler performance under load.** This is particularly apparent for clusters with large numbers of queriers.
```